app/controllers/users/service_provider_inactive_controller.rb

```diff
@@ -3,7 +3,7 @@ class ServiceProviderInactiveController < ApplicationController
     include FullyAuthenticatable
 
     def index
-      analytics.track_event(Analytics::SP_INACTIVE_VISIT)
+      analytics.sp_inactive_visit()
```

Comment:
I'd drop the optional parens here:
```suggestion
      analytics.sp_inactive_visit
```